org.civicrm.reqangular/src/common/services/file-upload.js

```diff
@@ -1,48 +1,65 @@
 define([
   'common/modules/services',
+  'common/angularFileUpload'
 ], function (module) {
   'use strict';
 
-  module.factory('FileUploadService', ['$q', 'FileUploader',
+  module.factory('FileUpload', ['$q', 'FileUploader',
     function ($q, FileUploader) {
-      var fileUploader;
+
+      /**
+      * Helper to call throw on expressions by wrapping it on function
+      *
+      * @param {String} error message to throw
+      */
+      function _throw (msg) {
```

Comment:
Also, this function as it is is a bit pointless, given that it's just calling `throw` with the given parameter (as in, what is it's purpose then? You might as well call `throw msg` directly

The way you could make it useful would be if it was something like

```js
function error(property) {
  throw property + 'missing from the uploader settings';
}
```